src/sort.ts

```diff
@@ -1,9 +1,17 @@
 import {AggregateOp} from 'vega';
 import {isArray} from 'vega-util';
 import {DateTime} from './datetime';
-import {VgComparatorOrder} from './vega.schema';
 
-export type SortOrder = VgComparatorOrder | null;
+export type SortOrder = 'ascending' | 'descending' | null;
+export type Compare =
```

Comment:
The first one is a `SortField`.  

Should `Compare = SortField | SortFields`? (Instead of redeclaring them?)

Reply:
Why is `ascending` and `descending` a sort field? It's a way to order.

Reply:
This is not related to SortOrder at all.  Here you have: 

```
Compare =
  | {
      field: string;
      order?: SortOrder;
    }
  | {
      field: string[];
      order?: (SortOrder)[];
    };
```

Reply:
Ok. Let's change this so we can merge.

Reply:
On it